web/docusaurus.config.js

```diff
@@ -1,11 +1,24 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
 const path = require('node:path');
 
+const tqlGrammar = require("@tenzir/vscode-tql/syntaxes/tql.tmLanguage.json");
+
 async function createConfig() {
+  const rehypePrettyCode = (await import('rehype-pretty-code')).default;
+  const shikiHighlighter = (await import('shiki')).createHighlighter;
+
+  const rehypePrettyCodeOptions = {
+    keepBackground: false,
+    theme: 'github-dark-dimmed',
```

Comment:
```suggestion
    theme: 'github-dark-default',
```

The contrast of `-dimmed` is very poor.

Reply:
Somewhat related, and I also already chatted with @dit7ya about this: We should try to get the light theme up and running for light mode docs, as that's what we embed in the Tenzir Platform as well.

Reply:
Changed the theme, but getting two themes working seems to be very hard without upgrading MDX.

Reply:
Okay, let's stick with the dark theme for now then and get back to that later. Can you write a follow-up story before merging this?

Reply:
I added it to this existing story. 

https://github.com/tenzir/issues/issues/1132